src/js/mixin/modal.js

```
@@ -86,168 +116,175 @@ export default {
-    }
 
-};
+            const timer = setTimeout(
+                () => {
+                    off();
+                    resolve();
+                },
+                toMs(css(transitionElement, 'transitionDuration')),
+            );
+        }),
+    ).then(() => delete el._reject);
```

Biome

[error] Avoid the delete operator which can impact performance.

Unsafe fix: Use an undefined assignment instead.


(lint/performance/noDelete)